src/lightly_train/_commands/export_onnx.py

```diff
@@ -0,0 +1,87 @@
+#
+# Copyright (c) Lightly AG and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# LICENSE file in the root directory of this source tree.
+#
+from __future__ import annotations
+
+import logging
+
+import torch
+
+from lightly_train import _logging
+from lightly_train._commands import _warnings, common_helpers
+from lightly_train._configs.config import PydanticConfig
+from lightly_train._task_models.dinov2_semantic_segmentation.dinov2_semantic_segmentation import (
+    DINOv2SemanticSegmentation,
+)
+from lightly_train.types import PathLike
+
+logger = logging.getLogger(__name__)
+
+
+def export_onnx(
```

Comment:
Let's call this `export_task` for now to allow different formats.

```suggestion
def export_task(
```

Reply:
I think later on we'll need the following arguments:
```
    *,
    out: PathLike,
    checkpoint: PathLike,
    format: Literal["onnx"],
    batch_size: int = 1,
    num_channels: int = 3,
    height: int = 224,
    width: int = 224,
    device: str = "auto",
    overwrite: bool = False,
    format_args: dict[str, Any] | None = None     # Format specific arguments. Eg. "dynamic" for onnx and int8 precision for tensorrt. 
```

Reply:
I don't think we need the `device` argument but the rest is necessary.